File: movie/movie-domain/src/main/java/com/project/domain/entity/Theater.java
package com.project.domain.entity;

import com.project.domain.common.BaseEntity;
import lombok.AccessLevel;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Entity
@Getter
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public class Theater extends BaseEntity {

}

File: movie/core/src/main/kotlin/com/example/movie/domain/theater/model/Theater.kt
package com.example.movie.domain.theater.model

import com.example.movie.domain.common.model.BaseModel
import java.time.LocalDateTime

class Theater(
    val id: Long,
    val name: String,
    override val createdBy: String,
    override val createdAt: LocalDateTime,
    override var updatedBy: String,
    override var updatedAt: LocalDateTime
) : BaseModel